__tests__/app.test.js

```diff
@@ -87,10 +87,42 @@ describe('get /api/articles/:article_id', ()=> {
                     votes: expect.any(Number)
             })
             expect(article.article_id).toBe(1)
-
         })
     })
 })
+describe('GET /api/articles/:article_id/comments', ()=> {
+    test('valid request -server responds with an array of comments objects', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(response => {
+            const comments = response.body.comments
+            expect(Array.isArray(comments)).toBe(true)
+            comments.forEach(comment => {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: expect.any(Number)
+                })
+            })
+        })
+    })
+    test('comments object array is ordered from most recent', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(response => {
```

Comment:
You may find an npm package called
[jest-sorted](https://www.npmjs.com/package/jest-sorted)   to be helpful here.

Reply:
hi Haz yeah Modou showed me yesterday but i cant get it to fail so i just popped  the old test back in for the sake of expediency until i can get some help with it